README.md

```diff
@@ -0,0 +1,124 @@
+| `model_name`                 | string                | **Required** |             | Vision model name as expected by the method [get_model()](https://pytorch.org/vision/main/models.html#listing-and-retrieving-available-models) from torchvision multi-weight API.                                                                                                                                                                  |
+| `weights`                    | string                | Optional     | `DEFAULT`   | Weights model name as expected by the method [get_model()](https://pytorch.org/vision/main/models.html#listing-and-retrieving-available-models) from torchvision multi-weight API.                                                                                                                                                                 |
+| `default_minimum_confidence` | float                 | Optional     |             | Default minimum confidence for filtering all labels that are not specified in `label_confidences`.                                                                                                                                                                                                                                                 |
+| `labels_confidences`         | dict[str, float]      | Optional     |             | Dictionary specifying minimum confidence thresholds for specific labels. Example: `{"grasshopper": 0.5, "cricket": 0.45}`. If a label has a confidence set lower that `default_minimum_confidence`, that confidence over-writes the default for the specified label if `labels_confidences` is left blank, no filtering on labels will be applied. |
+| `use_weight_transform`       | bool                  | Optional     | True        | Loads preprocessing transform from weights metadata.                                                                                                                                                                                                                                                                                               |
+| `input size`                 | List[int]             | Optional     | `None`      | Resize the image. Overides resize from weights metadata.                                                                                                                                                                                                                                                                                           |
+| `mean_rgb`                   | [float, float, float] | Optional     | `[0, 0, 0]` | Specifies the mean and standard deviation values for normalization in RGB order                                                                                                                                                                                                                                                                    |
```

Comment:
Torchvision don't guarantee that metadata would always carry preprocessing information. Also because they expect the community to add models. I made it more explicit in the README and pushed those optional configs at the very end.